src/ux/confirmInformation/ConfirmInformation.tsx

```diff
@@ -0,0 +1,111 @@
+// eslint-disable-next-line
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { SD } from '../../lib/sdr'
+import { Header2, SmallSpan, Span, Span2 } from '../../ui/shared/Typography'
+import { Button, ButtonSecondary } from '../../ui/shared/Button'
+import { credentialValueToText } from '../../vc-json-schema-adapter'
+import Loading from '../../ui/shared/Loading'
+import { Trans } from 'react-i18next'
+import i18next from 'i18next'
+import { getChainName } from '../../adapters'
+import { IProviderUserOptions } from 'web3modal'
+import Checkbox from '../../ui/shared/Checkbox'
+
+const DONT_SHOW_AGAIN_KEY = 'RLogin:DontShowAgain'
+
+interface ConfirmInformationProps {
+  chainId: number | undefined
+  address: string | undefined
+  sd: SD | undefined
+  providerUserOption: IProviderUserOptions
+  onConfirm: () => void
+  onCancel: () => void
+}
+
+export function ConfirmInformation ({ chainId, address, providerUserOption, sd, onConfirm, onCancel }: ConfirmInformationProps) {
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [dontShowAgainSelected, setDontShowAgainSelected] = useState<boolean>(false)
+  const data = sd ? Object.assign({}, sd.credentials, sd.claims) : {}
+
+  const handleSubmit = () => {
+    setIsLoading(true)
+    onConfirm()
+  }
+
+  useEffect(() => {
+    const DONT_SHOW_AGAIN_DEFAULT: boolean =
+      localStorage.getItem(DONT_SHOW_AGAIN_KEY) ? JSON.parse(localStorage.getItem(DONT_SHOW_AGAIN_KEY)!) : false
+
+    if (DONT_SHOW_AGAIN_DEFAULT) {
+      handleSubmit()
+    }
+  }, [])
+
+  return !isLoading
+    ? <>
+      <Header2><Trans>Information</Trans></Header2>
+      <CenterContent>
+        <ProviderLogo>
+          <img src={providerUserOption.logo} alt={providerUserOption.name} />
+        </ProviderLogo>
+      </CenterContent>
+
+      <div style={{ display: 'flex', padding: '50px 0', justifyContent: 'center', gap: '30px' }}>
+        <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-start' }}>
+          <Span><Trans>Wallet address</Trans>:</Span>
```

Comment:
I would suggest using a definition list here rather than columns with spans. This would be better for the user as it would be accessible, and better for the developer since it would be easier to read. 

I would suggest something like this:

```
<dl>
   <dt>Wallet address</dt>
   <dd>0x123...</dd>
   <dt>Netwrok</td>
   <dd>31</dd>
</dl>
```

I would also rename `Span` and `Span2` to be more descriptive. In this case, you could remove Span all together and use the existing dt (i.e. title) and the description with styled-components.